### PR TITLE
Made templates S3 bucket customizable

### PR DESCRIPTION
**Pull request by belminf**
* Template S3 bucket is now a parameter in root template.
* bin/deploy.sh script now accepts region and bucket name as arguments

### CODE REVIEW

File: bin/deploy [hunk 1]
```diff
@@ -13,28 +13,77 @@ regions=(
...
+      echo "ERROR: unknown parameter \"$PARAM\""
+      usage
+      exit 1
+      ;;
+  esac
+  shift
+done
+
+
+set -o xtrace
```
Comment (jpignata) [inline bin/deploy:62]: I could get behind killing xtrace entirely. Dubious of its value.

**Review by jpignata — APPROVED** [review]
LGTM. Feel free to merge it.

File: bin/deploy (hunk 1 above)
Comment (belminf) [reply bin/deploy:62]: Although having `xtrace` option on during args processing was too noisy, I could see some value in having it on during AWS CLI commands. Could be useful for the person running the command to understand what is going on.

That's why I moved it to L62.

Comment (jpignata) [reply bin/deploy:62]: 👍

**Comment by jpignata** [comment]
Feel free to reopen if you'd still like to merge this.

**Pull request closed without merging**